Backend/DataAccessLayer/Models/Session.cs

```diff
@@ -14,6 +14,13 @@ public Session()
             ExpiresAt = DateTime.UtcNow.AddMinutes(MINUTES_UNTIL_EXPIRATION);
             Id = Guid.NewGuid();
         }
+		public Session(DateTime CreatedAt, DateTime UpdatedAt, DateTime ExpiresAt)
```

Comment:
dont think you need this overloaded constructor, you can just set the values via accessor